server/src/DBService/ContactedPeople/Mutation.ts

```diff
@@ -7,3 +7,11 @@ mutation updateAllUnSavedContacts($unSavedContacts: JSON!) {
   }
 }
 `;
+
+export const CHECK_FOR_DUPLICATE_IDS = gql`
```

Comment:
I would call it "CHECK_FOR_DUPLICATE_CONTACTED_PERSON_IDS" to be clear that it checks contacted person ids and not just random ids.